usecases/optimization_paper/tex/tex_sections/calibration.tex

```diff
@@ -1,2 +1,103 @@
-Here is an empty file as example to start the calibration section.
-Feel free to create as many sections as necessary :D.
+Here is an empty file as example to start the calibration section.
+Feel free to create as many sections as necessary :D.
+
+\atul{30 Nov, 2022: !!Below is just initial documentation. .Please dont analyze the content yet. More will be added here.!!}
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Calibration -----------------------------------------
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection{Model Calibration}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Optimization -----------------------------------------
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\subsection{Optimization under uncertainty}
+
+
+
+
+
+
+\subsubsection{Non-differentiable objective/constraints with design variables as argument}
+%
+In lot of real world scenarios, complex computer simulators are used to build a relationship between parameters of the underlying theory to the experimental observations. Many a times the physics bases simulator/forward solver denoted by $\bm{y}(\cdot)$ is non-diffrentiable. For inference/optimization tasks involving these simulators, many active research areas are trying to tackle this. \cite{cranmer2020frontier, louppe_adversarial_2019, beaumont2002approximate,marjoram2003markov}. Elaborating further, if the simulator is related to the objective of some optimization problem and design variables $\bm{x}$ of the problem are direct input to the objective, then gradient based approaches are not directly applicable. The following optimization is desired:
+\begin{align}
+    \bm{x}^* = \min_{\bm{x}}\mathcal{O}(\bm{y}_o(\bm{x}))
```

Comment:
this is up to you, but it might be unseful to define your variables as macros, that way, it is simple to change the name of some variable.

Reply:
yes I will manage this dont worry